packages/discord.js/src/structures/MessagePayload.js

```diff
@@ -105,7 +105,7 @@ class MessagePayload {
     if (this.options.content === null) {
       content = '';
     } else if (typeof this.options.content !== 'undefined') {
-      content = Util.verifyString(this.options.content, RangeError, 'MESSAGE_CONTENT_TYPE', true);
+      content = Util.verifyString(this.options.content, RangeError, 'MessageContentType', true);
```

Comment:
Does this actually work?

Reply:
No, bad replace, but that is a thing I missed while changing things to use `ErrorCodes`

https://github.com/discordjs/discord.js/blob/f9dce573be139b047a05d82aab875371fb8611fd/packages/discord.js/src/util/Util.js#L365-L374

This is a bit dangerous right here - `message` can be a string just fine since `error` defaults to `globalThis.Error`, but it should be okay.